src/utils/isStandardSyntaxMediaFeature.js

```diff
@@ -16,5 +16,8 @@ export default function (mediaFeature) {
   // SCSS or Less interpolation
   if (hasInterpolation(mediaFeature)) { return false }
 
+  // SCSS variables
+  if (/\$.+?/.test(mediaFeature)) { return false }
```

Comment:
> it is very important

I don't think it is actually very important — it's sort of a wacky edge-case to use an escaped `\$` in an identifier. But if you'd like to fix it, go for it.